### ISSUE TITLE
Is the paper available?

### ISSUE THREAD

**Issue by hurueilin** [VTCC-uTVM/2022AICityChallenge-Track3#1]
Thanks for sharing your work with us! 
Is the paper available so that I can get the detail of your implementation?

**Comment by VTCC-uTVM** [comment]
Sorry for the delayed response. Our CVPR workshop paper will be released soon! Thank you.

**Comment by hurueilin** [comment]
> Sorry for the delayed response. Our CVPR workshop paper will be released soon! Thank you.

Thanks for replying!

BTW, I'm trying to reproduce the training process, but I've met some problem.
Where is the 'X3D_checkpoint_epoch_00002.pyth' file in X3D_L.yaml?
![圖片](https://user-images.githubusercontent.com/20622590/167301254-f808178d-800f-4e1e-b70e-c31ffcbf9fab.png)
I've tried commented out this line (does it means training from scratch?) and modified batch size from 8 to 2.
But after inferencing, I get low F1-score 0.0213 on the submission system.
I use two GTX 1080 Ti GPU for training.

Is there anything I've overlook in order to reproduce your training results? Does the learning rate or anything else in X3D_L.yaml need to be modified? Thank you.

**Comment by VTCC-uTVM** [comment]
We don't use CHECKPOINT_FILE_PATH. We used pretrain x3d_l_1.pyth, download in https://drive.google.com/drive/folders/1UwhvMoCf36y82KN5rd7ZHzrv1-aTWprl 
You can check in file https://github.com/VTCC-uTVM/2022AICityChallenge-Track3/blob/b25080f77446b7547b62389424272504a8a3c6f4/X3D_training/slowfast/utils/checkpoint.py#L322
Can you share your eval result in the training log?

**Comment by hurueilin** [comment]
Got it. I have downloaded 'x3d_l_1.pyth' and placed it in under X3D_training folder. 
So the CHECKPOINT_FILE_PATH (line 8) in X3D_L.yaml can be deleted? It will load 'x3d_l_1.pyth' pretrain file automatically. Is that correct?

The following are the training logs of 15 models:
[checkpoint_dashboard_24026_stdout.log](https://github.com/VTCC-uTVM/2022AICityChallenge-Track3/files/8649111/checkpoint_dashboard_24026_stdout.log)
[checkpoint_dashboard_24491_stdout.log](https://github.com/VTCC-uTVM/2022AICityChallenge-Track3/files/8649112/checkpoint_dashboard_24491_stdout.log)
[checkpoint_dashboard_35133_stdout.log](https://github.com/VTCC-uTVM/2022AICityChallenge-Track3/files/8649113/checkpoint_dashboard_35133_stdout.log)
[checkpoint_dashboard_38058_stdout.log](https://github.com/VTCC-uTVM/2022AICityChallenge-Track3/files/8649114/checkpoint_dashboard_38058_stdout.log)
[checkpoint_dashboard_49381_stdout.log](https://github.com/VTCC-uTVM/2022AICityChallenge-Track3/files/8649115/checkpoint_dashboard_49381_stdout.log)
[checkpoint_rearview_24026_stdout.log](https://github.com/VTCC-uTVM/2022AICityChallenge-Track3/files/8649116/checkpoint_rearview_24026_stdout.log)
[checkpoint_rearview_24491_stdout.log](https://github.com/VTCC-uTVM/2022AICityChallenge-Track3/files/8649117/checkpoint_rearview_24491_stdout.log)
[checkpoint_rearview_35133_stdout.log](https://github.com/VTCC-uTVM/2022AICityChallenge-Track3/files/8649118/checkpoint_rearview_35133_stdout.log)
[checkpoint_rearview_38058_stdout.log](https://github.com/VTCC-uTVM/2022AICityChallenge-Track3/files/8649119/checkpoint_rearview_38058_stdout.log)
[checkpoint_rearview_49381_stdout.log](https://github.com/VTCC-uTVM/2022AICityChallenge-Track3/files/8649120/checkpoint_rearview_49381_stdout.log)
[checkpoint_right_24026_stdout.log](https://github.com/VTCC-uTVM/2022AICityChallenge-Track3/files/8649121/checkpoint_right_24026_stdout.log)
[checkpoint_right_24491_stdout.log](https://github.com/VTCC-uTVM/2022AICityChallenge-Track3/files/8649122/checkpoint_right_24491_stdout.log)
[checkpoint_right_35133_stdout.log](https://github.com/VTCC-uTVM/2022AICityChallenge-Track3/files/8649123/checkpoint_right_35133_stdout.log)
[checkpoint_right_38058_stdout.log](https://github.com/VTCC-uTVM/2022AICityChallenge-Track3/files/8649124/checkpoint_right_38058_stdout.log)
[checkpoint_right_49381_stdout.log](https://github.com/VTCC-uTVM/2022AICityChallenge-Track3/files/8649125/checkpoint_right_49381_stdout.log)

Here's the  X3D_L.yaml config file I've used in this training:
https://drive.google.com/file/d/10eWFXYulHyXfuIwFguN16FebDTHuyCx9/view?usp=sharing

Is my training process similar to yours? Because I noticed that my loss only going down a little bit.
Thanks for your kindly reply again!

**Comment by VTCC-uTVM** [comment]
Don't comment this line https://github.com/VTCC-uTVM/2022AICityChallenge-Track3/blob/b25080f77446b7547b62389424272504a8a3c6f4/X3D_training/configs/Kinetics/X3D_L.yaml#L8
You should only change  batchsize and learning rate.

**Comment by hurueilin** [comment]
> Don't comment this line
> 
> https://github.com/VTCC-uTVM/2022AICityChallenge-Track3/blob/b25080f77446b7547b62389424272504a8a3c6f4/X3D_training/configs/Kinetics/X3D_L.yaml#L8
> 
> You should only change batchsize and learning rate.

The 'X3D_checkpoint_epoch_00002.pyth' file not found error occurs if I don't comment this line. 
![圖片](https://user-images.githubusercontent.com/20622590/167534101-32a0334f-e569-40b8-aa77-68f8a7e33eb6.png)

**Comment by VTCC-uTVM** [comment]
@hurueilin Thanks for your feedback. It is our mistake, we have fixed this bug in 3 commit: a9abaa8, 4050acd, f2be689.
Pretrained model, now can be download from official slowfast respository (Updated in readme).

**Comment by hurueilin** [comment]
> @hurueilin Thanks for your feedback. It is our mistake, we have fixed this bug in 3 commit: [a9abaa8](https://github.com/VTCC-uTVM/2022AICityChallenge-Track3/commit/a9abaa83bbfb0487b5eed66f1becbd27e6270495), [4050acd](https://github.com/VTCC-uTVM/2022AICityChallenge-Track3/commit/4050acd477a41e3f3b04a4e0a5510e09783f3fcd), [f2be689](https://github.com/VTCC-uTVM/2022AICityChallenge-Track3/commit/f2be6899469a2af3289b147dca44a7d356631ca2). Pretrained model, now can be download from official slowfast respository (Updated in readme).

Thanks! I'll train a new one and check if the performance becomes better.

**Comment by VTCC-uTVM** [comment]
Our paper is published here, [An Effective Temporal Localization Method With Multi-View 3D Action Recognition for Untrimmed Naturalistic Driving Videos](https://openaccess.thecvf.com/content/CVPR2022W/AICity/html/Tran_An_Effective_Temporal_Localization_Method_With_Multi-View_3D_Action_Recognition_CVPRW_2022_paper.html)

**Comment by hurueilin** [comment]
> Our paper is published here, [An Effective Temporal Localization Method With Multi-View 3D Action Recognition for Untrimmed Naturalistic Driving Videos](https://openaccess.thecvf.com/content/CVPR2022W/AICity/html/Tran_An_Effective_Temporal_Localization_Method_With_Multi-View_3D_Action_Recognition_CVPRW_2022_paper.html)

Thanks for updating. Really appreciate your work. 

Sorry for keep disturbing. I have one question about the Table 3 in your paper. 
Are these results evaluated on A2 dataset? Why both F1-scores are much lower than the final F1-score 0.3492? 
![圖片](https://user-images.githubusercontent.com/20622590/173780101-f085afae-25d1-4801-a122-08f24fe89a4e.png)

**Comment by VTCC-uTVM** [comment]
This result is evaluated on A2 dataset. F1-scores in table 3 isn't final result. This proved that our method (second method) is better than the old method (first method).
Final result is combined both multi-view, ensemble, hypermeter tuning and our post-processing.

**Comment by hurueilin** [comment]
> This result is evaluated on A2 dataset. F1-scores in table 3 isn't final result. This proved that our method (second method) is better than the old method (first method). Final result is combined both multi-view, ensemble, hypermeter tuning and our post-processing.

So the scores in Table 3 didn't use multi-view ensemble? (I'm interested in why there's a huge gap between these scores and final score.)
Can you specify more details in order to reproduce your best final result? The best score I can get now is F-score=0.289. 
Thanks for your replying as always.

**Comment by VTCC-uTVM** [comment]
Table 3 didn't use multi-view ensemble. Model training in the table uses number of frame is 15, frame rate is 4, however we infer with number of frame is 8, frame rate is 4. Final models is trained and inferred with number of frame is 8 frame rate is 4.
Table show that our post-processing better than old post-processing.
We trained model in  A100 GPU, so if you use GPU others, the result is different. You must tuning hyper-parameter appropriate with your GPU config.
